src/hotspot/share/oops/klass.cpp

```diff
@@ -386,6 +403,7 @@ uintx Klass::hash_secondary_supers(Array<Klass*>* secondaries, bool rewrite) {
       }
     }
     assert(i == secondaries->length(), "mismatch");
+    postcond((int)population_count(bitmap) <= secondaries->length());
```

Comment:
This helps explain my confusion reading fallback_search_secondary_supers().  Could we refer to this invariant there?  And could this be strengthened to :
`population_count(bitmap) == secondaries->length() || (population_count(bitmap) == SECONDARY_SUPERS_TABLE_SIZE && secondaries->length() > SECONDARY_SUPERS_TABLE_SIZE)`
?

Reply:
We can't assert that here, because at this point we know `length <= SECONDARY_SUPERS_TABLE_SIZE - 2`.

Reply:
Doesn't that mean `population_count(bitmap) == secondaries->length()`?